ipapython/ipaldap.py

```diff
@@ -1085,7 +1085,7 @@ def simple_bind(self, bind_dn=DN(('cn', 'directory manager')),
             self.conn.simple_bind_s(
                 bind_dn, bind_password, server_controls, client_controls)
 
-    def external_bind(self, user_name, server_controls=None,
+    def external_bind(self, user_name=None, server_controls=None,
```

Comment:
Remove the `user_name` argument altogether, as it always has to be set to the effective username.